src/Dict/Any.elm

```diff
@@ -573,3 +573,47 @@ encode keyE valueE =
     Encode.object
         << List.map (Tuple.mapBoth keyE valueE)
         << toList
+
+
+{-| Turn an AnyDict into a JSON list of tuples. This is useful when you have more complex types as keys
+
+    import Json.Decode as Decode
+    import Json.Encode as Encode
+
+    type alias Person = {first : String, last : String}
+    type alias Age = Int
+
+    personToString : Person -> String
+    personToString {first, last} = first ++ last
```

Comment:
it would be safer to use pair of String for the purpose of comparable type in here I think.